Time for Coffee! WatchOS 2 App Extension/ComplicationController.swift

```diff
@@ -0,0 +1,225 @@
+                    }
+                    
+                handler(entries)
+                }
+            }
+        }
+        TFCWatchData.sharedInstance.getStations(handleReply, errorReply: nil, stopWithFavorites: true)
+    }
+    
+    // MARK: - Update Scheduling
+    
+    func getNextRequestedUpdateDateWithHandler(handler: (NSDate?) -> Void) {
+        // Call the handler with the date when you would next like to be given the opportunity to update your complication content
+        let nextUpdateDate = NSDate().dateByAddingTimeInterval(Constants.FrequencyOfTimelineUpdate) // request an update each 1.5 hour
+        handler(nextUpdateDate);
+    }
+    
+    // MARK: - Placeholder Templates
+    
+    func getPlaceholderTemplateForComplication(complication: CLKComplication, withHandler handler: (CLKComplicationTemplate?) -> Void) {
+        
+        switch complication.family {
+            
+        case CLKComplicationFamily.ModularLarge:
+            let tmpl = CLKComplicationTemplateModularLargeTable()
+            
+            tmpl.headerTextProvider = CLKSimpleTextProvider(text: "Station")
+            tmpl.row1Column1TextProvider = CLKSimpleTextProvider(text: "Line - Destination", shortText: nil)
+            tmpl.row1Column2TextProvider = CLKSimpleTextProvider(text: "Time", shortText: nil)
+            tmpl.row2Column1TextProvider = CLKSimpleTextProvider(text: "Line - Destination", shortText: nil)
+            tmpl.row2Column2TextProvider = CLKSimpleTextProvider(text: "Time", shortText: nil)
+            
+            handler(tmpl)
+            
+        default: break
+        }
+    }
+    
+    func requestedUpdateDidBegin() {
+        // get the shared instance
+        let server = CLKComplicationServer.sharedInstance()
+        
+        // reload the timeline for all complications
+        for complication in server.activeComplications {
+            server.reloadTimelineForComplication(complication)
+        }
+    }
+    
+    func requestedUpdateBudgetExhausted() {
+        // get the shared instance
+        let server = CLKComplicationServer.sharedInstance()
+        
+        // reload the timeline for all complications
+        for complication in server.activeComplications {
+            server.reloadTimelineForComplication(complication)
+        }
+    }
+    
+    //MARK: - Convenience
+    
+    private func templateForStationDepartures(station: TFCStationBase , departure: TFCDeparture, nextDeparture: TFCDeparture?, complication: CLKComplication) -> CLKComplicationTemplate {
+        
+        let tmpl = CLKComplicationTemplateModularLargeTable() // Currently supports only ModularLarge
+        
+        tmpl.headerTextProvider = CLKSimpleTextProvider(text: station.getName(true))
+        let departureLine = departure.getLine()
+        let nextDepartureLine = nextDeparture?.getLine() ?? "-"
+
+        let nextDepartureDestination = nextDeparture?.getDestination() ?? "-"
```

Comment:
You can use nextDeparture?.getDestination(station) to have the localName part first (if the city matches the station city), so that it's clearer on the watch